doc/specs/vulkan/appendices/VK_GOOGLE_display_timing.txt

```diff
@@ -28,8 +28,8 @@ They need to know when presentable images were actually presented, and when
 they could have been presented.
 Applications also need to tell the presentation engine to display an image
 no sooner than a given time.
-This can allow the application's animation to look smooth to the user, with
-no stuttering.
+This can allow the application's animation to be without stuttering, so it
```

Comment:
"to avoid stuttering" would probably flow better?

Reply:
agreed